webpack.config.ts

```diff
@@ -1,45 +1,52 @@
 /// <reference types="./source/globals" />
 
 import path from 'path';
-import {readdirSync, readFileSync} from 'fs';
+import {readFileSync} from 'fs';
 
 import SizePlugin from 'size-plugin';
-import stripIndent from 'strip-indent';
 // @ts-expect-error
 import {ESBuildPlugin} from 'esbuild-loader';
 import CopyWebpackPlugin from 'copy-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import webpack, {Configuration} from 'webpack';
 
-function parseFeatureDetails(id: FeatureID): FeatureMeta {
-	const content = readFileSync(`source/features/${id}.tsx`, {encoding: 'utf-8'});
-	const fields = ['disabled', 'description', 'screenshot'] as const;
+import concatRegex from './source/helpers/concat-regex';
 
-	const feature: Partial<FeatureMeta> = {id};
-	for (const field of fields) {
-		const value = new RegExp(`\n\t${field}: '([^\\n]+)'`).exec(content)?.[1];
-		if (value) {
-			const validValue = value.trim().replace(/\\'/g, '’'); // Catch trailing spaces and incorrect apostrophes
-			if (value !== validValue) {
-				throw new Error(stripIndent(`
-					❌ Invalid characters found in \`${id}\`. Apply this patch:
+function parseFeatureDetails(readmeContent: string, id: FeatureID): FeatureMeta {
+	const lineRegex = concatRegex(/^- \[]\(# "/, id, /"\)(?: 🔥)? (.+)$/m);
+	const lineMatch = lineRegex.exec(readmeContent);
+	if (lineMatch) {
+		const urls: string[] = [];
 
-					- ${field}: '${value}'
-					+ ${field}: '${validValue}'
-				`));
-			}
+		return {
+			id,
+			description: lineMatch[1].replace(/\[(.+?)]\((.+?)\)/g, (_match, title, url) => {
+				urls.push(url);
+				return title;
+			}),
+			screenshot: urls.find(url => /\.(png|gif)$/i.test(url))
+		};
+	}
 
-			feature[field] = value.replace(/\\\\/g, '\\');
-		}
+	// Feature might be highlighted in the readme
+	const imageRegex = concatRegex(/<p><a title="/, id, /"><\/a> (.+?)\n\t+<p><img src="(.+?)">/);
+	const imageMatch = imageRegex.exec(readmeContent);
+	if (imageMatch) {
+		return {
+			id,
+			description: `${imageMatch[1]}.`,
```

Comment:
Template literals are too noisy if you're just appending text
```suggestion
			description: imageMatch[1] + '.',
```